addons/clones/addon.json

```diff
@@ -26,6 +30,12 @@
     }
   ],
   "settings": [
+    {
+      "id": "editoronly",
+      "name": "Only show in editor",
```

Comment:
Would a show on project page setting be better?

Reply:
Maybe, since "only show in editor" is closer to implying that having the clone counter only appear in the editor is not the default option.

Reply:
I agree. The opposite sounds more intuitive. It feels weird when enabling something removes functionality.